### PR TITLE
Add "command_exec" operation

### PR DESCRIPTION
**Pull request by sky003**

### CODE REVIEW

File: capysvc/service_definition.go [hunk 1]
```diff
@@ -1154,3 +1157,98 @@ func (o *Operation) newFilesystemInputRemoveOperation() (*operations.FilesystemI
...
 }
+
+func (o *Operation) newCommandExecOperation(ctx Context) (*operations.CommandExecOperation, error) {
+	parameterLoaderProvider, providerErr := ctx.ParameterLoaderProvider()
+	if providerErr != nil {
+		return nil, providerErr
+	}
+
+	var commandName string
+	if commandNameParameter, ok := o.Params["commandName"]; ok {
```
Comment (matiasinsaurralde) [inline capysvc/service_definition.go:1168]: Probably better and more legible to return earlier in the opposite condition like:
```go
if commandNameParameter, ok := o.Params["commandName"]; !ok {
  return nil, errors.New("failed to retrieve \"commandName\" parameter")
}
parameterLoader, loaderErr := parameterLoaderProvider.ParameterLoader(
...
```

File: capysvc/service_definition.go [hunk 2]
```diff
@@ -1154,3 +1157,98 @@ func (o *Operation) newFilesystemInputRemoveOperation() (*operations.FilesystemI
...
+			return nil, valErr
+		}
+
+		commandName = val
+	} else {
+		return nil, errors.New("failed to retrieve \"commandName\" parameter")
+	}
+
+	var commandArgs []string
+	if commandArgsParameter, ok := o.Params["commandArgs"]; ok {
```
Comment (matiasinsaurralde) [inline capysvc/service_definition.go:1188]: Same as the previous comment.

File: capysvc/service_definition.go [hunk 3]
```diff
@@ -1154,3 +1157,98 @@ func (o *Operation) newFilesystemInputRemoveOperation() (*operations.FilesystemI
...
+			return nil, valErr
+		}
+
+		commandArgs = val
+	} else {
+		return nil, errors.New("failed to retrieve \"commandArgs\" parameter")
+	}
+
+	var outputFileDestination string
+	if outputFileDestinationParameter, ok := o.Params["outputFileDestination"]; ok {
```
Comment (matiasinsaurralde) [inline capysvc/service_definition.go:1208]: Same here

File: capysvc/service_definition.go [hunk 4]
```diff
@@ -1154,3 +1157,98 @@ func (o *Operation) newFilesystemInputRemoveOperation() (*operations.FilesystemI
...
+			return nil, valErr
+		}
+
+		outputFileDestination = val
+	} else {
+		return nil, errors.New("failed to retrieve \"outputFileDestination\" parameter")
+	}
+
+	var allowParallelExecution bool = false
+	if allowParallelExecutionParameter, ok := o.Params["allowParallelExecution"]; ok {
```
Comment (matiasinsaurralde) [inline capysvc/service_definition.go:1228]: And here.

File: capysvc/service_definition.go (hunk 1 above)
Comment (sky003) [reply capysvc/service_definition.go:1168]: I actually tried this and it introduced even more complexity that is now related to the scope of the variables. So it doesn't seem to work well in this case.

**Pull request merged**